extensions/scarb-doc/src/docs_generation/markdown/traits.rs

```diff
@@ -59,6 +61,36 @@ pub trait MarkdownDocItem: DocItem {
         header_level: usize,
     ) -> Result<String>;
 
+    fn get_short_documentation(&self, context: &MarkdownGenerationContext) -> String {
+        let mut short_doc_buff = String::new();
+
+        if let Some(tokens) = self.doc() {
+            for token in tokens.iter() {
+                match token {
+                    DocumentationCommentToken::Content(content) => {
+                        if SHORT_DOCUMENTATION_AVOID_PREFIXES
+                            .iter()
+                            .any(|prefix| content.starts_with(prefix))
```

Comment:
Won't `"\n\n"` be separated into two separate tokens though? 🤔 Taking it from https://github.com/FroyaTheHen/cairo/blob/0b867e8b17690f8963f5722bb3dab143a7e66760/crates/cairo-lang-doc/src/tests/test-data/lists_formatting.txt#L131-L132